### PR TITLE
Feature cli exit codes

### PR DESCRIPTION
**Pull request by adamc00**
I wanted to have ally exit with an error code (non zero) upon audit failures. This PR adds that feature. I'm not a native node dev so please don't hesitate to provide feedback to make this PR acceptable if it is not.

### CODE REVIEW

**Comment by adamc00** [comment]
I'll fix these issues, add a test and try again!

**Comment by sindresorhus** [comment]
It should just do that by default. No option needed. IMHO.

File: cli.js [hunk 1]
```diff
@@ -68,6 +76,7 @@ eachAsync(urls, function (url, i, next) {
             }
 
             if (el.result === 'FAIL') {
+                exitCode = 1;
```
Comment (sindresorhus) [inline cli.js]: `process.exitCode = 1;` and you can remove the above.

**Comment by addyosmani** [comment]
Friendly ping on feedback @adamc00

**Comment by adamc00** [comment]
I have closed this PR and will make a new clean one which simply implements non zero exit code upon failure(s).

**Pull request closed without merging**